src/service/core/vmHandler.go

```diff
@@ -285,3 +280,67 @@ func (dm *KVMS) HandleVMLabels(event tp.KubeArmorVirtualMachineLabel) string {
 func (dm *KVMS) HandleNetworkPolicyUpdates(req *cilium.NetworkPolicyRequest) {
 	cilium.HandlePolicyUpdates(dm.EtcdClient, req)
 }
+
+func (dm *KVMS) RestoreFromEtcd() error {
+	// 1. Restore MapIdentityToEWName
+	id2Ew, err := dm.EtcdClient.EtcdGet(context.Background(), ct.KvmOprIdentityToEWName)
+	if err != nil {
+		kg.Errf("Failed to retrieve information from etcd key prefix %s", ct.KvmOprIdentityToEWName)
```

Comment:
```suggestion
		kg.Errf("Failed to retrieve information from etcd key prefix %s, err=%s", ct.KvmOprIdentityToEWName, err.Error())
```

Reply:
We're returning the `err` to the parent function and logging it there, along with the restoration-failed message. Do we have to log at this function also?

(same applies to all the comments below)